community/gardening/index.md

```diff
@@ -4,31 +4,22 @@ title:  "Spinnaker Gardening Days #communityHack"
 sidebar:
   nav: community
 ---
-_Welcome to the command center for Spinnaker Gardening Days, a community celebration and hackathon._
-{%
-  include
-  figure
-  image_path="./Spinnaker-Garden-1C.jpg"
-%}
+![image](SpinnakerGardeningDays.jpg)
 ## Register
-[Register for Spinnaker Gardening Days](https://www.eventbrite.com/e/spinnaker-gardening-days-communityhack-tickets-97845696111) on Eventbrite to reserve your tickets.
+[Register for Spinnaker Gardening Days](https://go.armory.io/gardening) to join our hackathon and earn rewards for your Spinnaker contributions July 16-23.
 
 ## Jump In!
-To attend Spinnaker Gardening Days, you'll need Zoom. See Zoom's ["Joining a Meeting" documentation](https://support.zoom.us/hc/en-us/articles/201362193-Joining-a-Meeting) to learn how to join.
-
-* Join the [__Gardening Central__ Zoom Room](https://armory.zoom.us/j/6807216019) to enter Spinnaker Gardening Days
-* Join [Spinnaker Slack](https://join.spinnaker.io) to chat in [event & team channels](slack-map.md). Visit __#gardening-general__, our main event channel.
-* [Master schedule](schedule.md)
-* [Slack map](slack-map.md)
-* [What to hack](what-to-hack.md)
-* [Hackathon logistics](hack-logistics.md)
-* [Development environments](dev-environment.md)
-* [Developing plugins](developing-plugins.md)
-* [Read about prizes](prizes.md)
-* [Spinnaker-hackathon Github organizaion](https://github.com/spinnaker-hackathon)
-* [Spinnaker Gardening repository](https://github.com/spinnaker-hackathon/gardening)
+* Join [Spinnaker Slack](https://join.spinnaker.io) to chat in [event & team channels](slack-map.md). Visit __#gardening-general__, our main event channel to engage
```

Comment:
```suggestion
* Join [Spinnaker Slack](https://join.spinnaker.io) to chat in [event & team channels](slack-map.md). Visit __#gardening-general__, our main event channel to engage.
```

Reply:
End each of these list items with a period, since they're all complete sentences.